AK/StdLibExtras.h

```diff
@@ -141,20 +141,21 @@ constexpr bool is_constant_evaluated()
     constexpr type abs(type num)                    \
     {                                               \
         if (is_constant_evaluated())                \
-            return num < zero ? -num : num;         \
-        else                                        \
-            return __builtin_##intrinsic(num);      \
+            return num < (zero) ? -num : num;       \
+        return __builtin_##intrinsic(num);          \
     }
 
 __DEFINE_GENERIC_ABS(int, 0, abs);
-__DEFINE_GENERIC_ABS(long, 0l, labs);
-__DEFINE_GENERIC_ABS(long long, 0ll, llabs);
+__DEFINE_GENERIC_ABS(long, 0L, labs);
+__DEFINE_GENERIC_ABS(long long, 0LL, llabs);
 #ifndef KERNEL
-__DEFINE_GENERIC_ABS(float, 0.0f, fabsf);
+__DEFINE_GENERIC_ABS(float, 0.0F, fabsf);
 __DEFINE_GENERIC_ABS(double, 0.0, fabs);
-__DEFINE_GENERIC_ABS(long double, 0.0l, fabsl);
+__DEFINE_GENERIC_ABS(long double, 0.0L, fabsl);
```

Comment:
These changes look like they should belong to a separate "fix literal suffixes" commit, instead of this "Avoid else after return" commit.

Reply:
All of these are clang-tidy fixes of various types; most of the time you fix warnings you'll find one type that dominates most others, but splitting them up is a lot of work for the submitter.

Reply:
I spent a lot of time with the S key in git add -p 🙃. I can split this up if it makes sense though, it is kinda different from the other fix in this file